addons/obs-websocket-gd/obs_websocket.gd

```diff
@@ -998,13 +998,12 @@ func _get_message() -> Dictionary:
 	var message: String = obs_client.get_peer(1).get_packet().get_string_from_utf8().strip_edges()
 
 	var test_json_conv = JSON.new()
-	test_json_conv.parse(message)
-	var json_result: JSON = test_json_conv.get_data()
-	if json_result.error != OK:
+	var error: Error = test_json_conv.parse(message)
+	if error != OK:
 		logger.error("Unable to parse Hello from obs-websocket: %s\nAborting connection" % message)
 		return {}
 
-	var result = json_result.result
+	var result =  test_json_conv.get_data()
```

Comment:
I don't know if you want this typed to a Variant since we're just going to type check it in a moment

Reply:
You are right, not assigning a type is fine here.